cinema_service/urls.py

```diff
@@ -2,7 +2,8 @@
 from django.urls import path, include
 
 urlpatterns = [
-    path("admin/", admin.site.urls),
+    path("api/admin/", admin.site.urls),
     path("api/cinema/", include("cinema.urls", namespace="cinema")),
```

Comment:
The 'include' function for 'cinema.urls' uses a namespace. Ensure that the 'cinema.urls' module defines an app_name variable to support namespacing. If not, you should remove the namespace argument.